lib/ansible/modules/packaging/os/redhat_subscription.py

```diff
@@ -234,16 +238,22 @@ def unsubscribe(self):
 
     def update_plugin_conf(self, plugin, enabled=True):
         plugin_conf = '/etc/yum/pluginconf.d/%s.conf' % plugin
+
```

Comment:
I will do this in a follow up PR. Wanted to simply address this issue with this PR.